.travis.yml

```diff
@@ -1,10 +1,7 @@
-# While Travis Support has been implemented for OSX, it does not fully build
-# due to the build time limit and the vast ammount of warnings generated
-
 os:
+  - linux
```

Comment:
At this moment we are using SemaphoreCI because TravisCI was too slow, but if it's working now we could move there...